### ISSUE TITLE
VDD power delivery.

### ISSUE THREAD

**Issue by evanboonie** [dosdude1/MPx-CPU-Board-744x#1]
VDD power delivery will need to be designed for the CPUs. The input voltage will be 5v. The buck converter must at least be able to support output voltages from 1.15V to 1.95V to support the full range of 7445, 7447, and 7448 CPUs. The 7448 CPU will consume the most power. The 7448 data sheet only lists power draw up to 1.7Ghz. At 1.3v and 1.7Ghz, a 7448 will pull up to 29.8W if all execution units are fully utilized. If we extrapolate from lower frequency power draw, a 7448 running at 2Ghz could pull up to 34W. Therefore, if we plan to run dual 2Ghz 7448 CPUs, VDD must be able to supply at least 53A. We should design in some headroom so that we are not pushing the thermal limits of the buck converter when the CPUs are fully loaded. I would suggest an output current max of at least 60A to be safe.

To make it easy to support multiple CPUs and allow user under/over-volting, the buck regulator should be able to be set using VID/PVID with resistors programming the registers. There are not many buck converters/controllers that are still manufactured using VID/PVID to set their output voltage that also meet our requirements. However, there are several VID/PVID "programmers" on the market which can drive the feedback pin of many other modern buck converters. It might be worth exploring such VID programmers to open up more options for the design of the VDD rail.

**Comment by dosdude1** [comment]
So far, the best thing I've been able to find (that's readily-available, at least) is the Intersil ISL62882, which supports up to 60A of current, and has VID control. As a bonus, some MacBook models use this as the GPU VCORE regulator, so there are a plethora of donor boards available if one needs to be sourced that way. The only issue with it is it only goes up to 1.5V, though from what I've seen, the highest voltage any 7445 CPU uses is 1.57V (this is for the best 1.25 GHz version available, verified with iMac G4 schematic), and I don't think going down to 1.5V in the case of a 7445 at rated clock speed would cause any problems. The 7447 and 7448 only need 1.3V, so no issue with those at all. https://www.mouser.com/ProductDetail/Renesas-Intersil/ISL62882HRTZ

Alternatively, I also found the Analog Devices LTC3732, which also supports 60A max, but goes up to 1.85V. This regulator also has VID control, but is 3-phase, so may require more MOSFETs to use. Also is not as easy to source. https://www.mouser.com/ProductDetail/Analog-Devices/LTC3732CGTRPBF

**Comment by evanboonie** [comment]
Looking into it more, I think you are correct that going anywhere above 1.6v won't have much purpose even if the 7445 technically supports 1.95v. I'm not much of an overclocker, but I do doubt that the 7445 could be pushed beneficially far with those higher voltages.

I did a bit more digging today and found some more options that are currently in production.

Single Phase:
- [TI TPS51215A](https://www.ti.com/product/TPS51215A)
  -  Pros:
     - 1.8 million available for order now direct from TI (TI is very friendly and easy for direct orders compared to most manufacturers)
     - Very inexpensive at $1.112 for small quantities
  - Cons:
    - Only allows 4 voltage options, but they can be set to any value desired (likely 1.2, 1.3, 1.4, and 1.5V in our case)
    - Datasheet does not show efficiency beyond 10A. The product page suggests a max of 30A, but this limit is nowhere to be found on the datasheet.
- [ON ADP3211](https://www.onsemi.com/products/power-management/dc-dc-power-conversion/controllers/adp3211)
  - Pros:
    - 1,700 available from Mouser
    - Granular 12.5mV steps (Every voltage we care about has bit5 and bit6 set to 0, so we would have 5 VID resistors)
    - Quite inexpensive at $1.59 (on Mouser)
  - Cons
    - Only goes up to 1.5V which limits 7445 overclocking potential
    - Datasheet only shows efficiency up to 15A

Two Phase:
- [Maxim/Analog MAX17030](https://www.analog.com/en/products/max17030.html)
  - Pros:
    - Granular 12.5mV steps (Every voltage we care about has bit5 and bit6 set to 0, so we would have 5 VID resistors)
    - Ability to add a third phase if we need it
  - Cons:
    - Only goes up to 1.5v which limits 7445 overclocking potential
    - Relatively expensive at $11.35 (on DigiKey)
    - Potentially limited availability. Only 13 in stock at DigiKey with 3,600 as "Factory stock". None available at Mouser.
- [Linear/Analog LTC1709-85](https://www.analog.com/en/products/ltc1709-85.html)
  - Pros:
    - Excellent voltage range of 1.05-1.85V
    - Relatively granular 25mV steps
    - 5 VID resistors
  - Cons:
    - Relatively expensive at $11.01 (on Mouser)
    - Relatively limited stock at 135 available (on Mouser, none on DigiKey)
    - Somewhat limited max switching frequency of 300kHz
    - Datasheet doesn't show efficiency beyond ~20A, but it looks like a bad trajectory
- [TI TPS53624](https://www.ti.com/product/TPS53624)
  - Pros:
    - Nearly 8,000 in stock directly from TI
    - Relatively inexpensive at $3.226
    - Also supports single phase operation
    - Relatively wide output voltage range of 0.03125-1.6V
    - Extremely granular 6.25mV steps
  - Cons:
    - The extremely granular voltage options mean we will end up with 7 VID resistors
    - The datasheet only shows efficiency up to 50A. The product page also suggests a limit of 50A, but the limit is not directly mentioned in the datasheet. While efficiency is trending down at 50A, I believe that with proper external component selection, that this controller will be able to deliver the 53A that we need.

If we went with a single phase controller, we would almost certainly end up with multiple FETs in parallel, as there really aren't that many that meet our requirements in a single package. By going with two phase, we likely wouldn't be increasing the number of FETs that the VDD supply would require. Going with multi-phase also opens up a lot of options for dual half bridge FET packages where two FETs are in the same package, which could actually reduce the amount of chips on the PCB and simplify trace routing.

A single phase controller will also require a very high-current inductor, which tend to be physically larger than smaller current capacity ones. Having physically shorter inductors will help with heatsink clearance. In the case of the Cube, for example, with an unmodified heatsink, there is only ~4.2mm of clearance above the board. If the stock mounting plate is knocked off and a custom one is used (or we manage to line the inductor up with the stock hole), that increases to ~8.2mm. The only real downside to two phase that I see is the requirement of an additional inductor and a few supporting components. Two phase will also result in a smoother voltage supply and likely improved transient response.

Therefore, out of the options above, I think I personally like the TI TPS53624 the best. The fact that it can also run in single phase opens up a lot of options for us. We could, for example, only populate half of the components on a single CPU board to reduce the cost of production with the option of upgrading in the future. Or, if routing two CPUs proves too difficult and we only have a single CPU board as a result of this effort, the same VDD rail design can still be utilized by simply eliminating the components on the second phase.

**Comment by dosdude1** [comment]
Sounds fine to me. I'd probably say optimally go for a two-phase design (as that's what Sonnet did with that dual-7447 upgrade board shown in the MacRumors thread). Hopefully you can find an inductor that is slim enough to fit in the Cube with its stock cooling system, but that can be worked around with a custom mounting plate relatively easily. I have attached an image here of that Sonnet card fully assembled, so you can use its design as a reference if needed.

![IMG_20220125_064709](https://github.com/dosdude1/MPx-CPU-Board-744x/assets/23435713/8acf2649-cab3-43c1-ad26-0fe767b6a6fa)

**Comment by evanboonie** [comment]
I think I've figured out a good MOSFET and inductor to use.

For the MOSFET, I found the [TI CSD86360Q5D](https://www.ti.com/product/CSD86360Q5D). It includes both FETs in a single package. The datasheet really does all of the hard math involved in calculating power loss too, which saves a ton of time. They appear to be incredibly easy to source, with TI, Mouser, and DigiKey all having thousands in stock. Based on our max power draw, they will need a decent bit of copper under them (or a connection to the heatsink) to effectively dissipate the heat, but the power loss on these is quite a bit better than some others I investigated in a similar format.

As for the inductor, I found the Bourns SRP1038CC-R22M at DigiKey [here](https://www.digikey.com/en/products/detail/bourns-inc/SRP1038CC-R22M/21263965). Availability isn't amazing, but I was able to find several others with the same footprint at the same specs. The Bourns just had the best price/performance ratio. It is also only 4mm tall, meaning it will clear the heatsink in a cube as long as they aren't in line with the lower part that contacts the CPU die.

With that out of the way, I think we have the components for the core of our rail picked out. Any chance you have a picture of the back side of that CPU card? It might be helpful in figuring out the capacitor configuration. The datasheets for both the TI TPS53624 and Intersil ISL62882 (found out that this is actually quite similar to the TI part) are pretty light on that topic. The TI datasheet just tells you to look at the CPU spec but to ballpark 1600μF and the 7448 datasheet effectively just says "several 100–330 μF tantalum caps with low ESR". None of the datasheets talk about input capacitance much at all either. I know some formulas that can help out, but seeing what Sonnet did would also be a nice reference.

**Comment by dosdude1** [comment]
Sure, attached here is an image of the backside of the assembled board. Unfortunately this is the best quality image of the back I have. Can get the same image of the unassembled one if needed.

With regard to parts, find as much as you can available on [LCSC](https://www.lcsc.com/). I intend to have JLCPCB manufacture and do SMD assembly on these boards (as manually assembling these would be extremely tedious and time-consuming). JLC uses LCSC as their parts supplier for assembly. I don't expect them to have all the parts available, such as the buck regulator, and maybe MOSFETs, but those I can solder manually without issue.

![IMG_20220125_072134](https://github.com/dosdude1/MPx-CPU-Board-744x/assets/23435713/79f9d9a3-b645-4665-847e-04cc707b73b1)

**Comment by dosdude1** [comment]
Also, here are a couple images that I have where you can actually read the capacitor values.

![IMG_20220125_072407](https://github.com/dosdude1/MPx-CPU-Board-744x/assets/23435713/8c4ab235-1a13-42c9-a5ac-0294b3a2c767)

![IMG_20220125_072401](https://github.com/dosdude1/MPx-CPU-Board-744x/assets/23435713/1f83e43e-8580-43e1-b8a9-f87fddaf0583)

**Comment by evanboonie** [comment]
Thanks for the photos. That is actually far more capacitance than I expected. Looks like 884uF of tantalum on the input side and 4,700uF on the output side. I got out one my my Cisco NPE-G2 boards out and they use 940uF of tantalum on the input and 3,670uF on the output for a single 7448. Not that it is helpful for automated assembly, but if someone wanted to be as cost effective and/or environmentally friendly as possible, harvesting the caps off of the NPE-G2 might be a halfway decent solution. Especially considering that NPE-G2s are the cheapest source I know of for 7448s. This many tantalum caps will probably be as expensive as everything else on the board combined outside of the CPUs themselves.

Checking LCSC, they do carry both of the TI parts that I'd like to use, but currently have none in stock. They have several inductors in the same values and footprint we need, but all from Chinese brands that I've never heard of. Probably fine as long as they aren't lying about their tolerance specs though. I'll probably build out the BOM for the capacitors and resistors on both DigiKey and LCSC (trying to find components in common where I can). On the topic of assembly, I can't guarantee anything right now, but I am seriously considering buying my own pick and place machine; so I might be able to help out with SMT assembly when it comes time to take that step.

**Comment by dosdude1** [comment]
I'm not too concerned about the price of the caps, I'd rather just get new ones than salvage some from other boards.

Also, someone brought up on the MacRumors thread about some Cube CPU upgrades having a way to produce CPU VCORE off the main 24V rail coming off the external PSU, by way of connecting external wires to the CPU board, rather than the usual 5V coming off the VRM board (which is a known weak point of the Cube). Not sure if that buck regulator or FETs support doing such a thing, or how to best implement that, but may be worth looking into.

I also may have a chance of getting a nice pick and place machine, but that is far out and not guaranteed, so I'd rather plan to just have JLC do it... Their pricing is not too bad. As mentioned, soldering a few things myself, such as the MPx connector, CPUs, buck regulator, etc. is no big deal, so no need to find every single thing on LCSC.

**Comment by evanboonie** [comment]
After doing more research into the power delivery for CPUs, I definitely agree that new caps are ideal. One of the reasons that I believe that the Sonnet board uses so many on the input is to lower the ESR to a point that the caps can handle the input ripple current. I don't see any ceramic caps on the input on their design and the best solid tantalum caps that are easily available have an ESR of 70mO. The Cisco design uses ceramics and only 2 tantalum on the input side which is a far more typical design and one that I definitely believe that we should pursue. Nowadays, compact aluminum polymer caps are probably the best for this type of application due to their extremely low ESR and better tolerance to abuse compared to tantalum designs. Based on what I could determine, the Sonnet card uses aluminum polymer on the output.

As for the voltage switching, the Cube external PSU actually runs at 28v. The buck controller I am using (along with all "IMVP-6.5" controllers as far as I can tell) can operate up to 28v. The FETs I had picked out only go up to 25v, but I did find one in the same package that supports up to 30v, just with slightly more power loss. So it is possible for the converter to support it, but there are definitely some trade-offs involved there. The biggest one will be the huge jump in required voltage on the input caps. There are far less that meet our spec at > 28V. It would definitely eliminate compact al-poly which will increase our ESR and require more total capacitors.

As the input voltage goes up, the efficiency of the converter will tend to fall, resulting in more power loss; likely pushing our MOSFETs too far to handle. Right now, based on my calculations, our FETs have about twice the conduction loss that they do switching loss, so putting two in parallel on each phase is definitely an option. In fact, I was actually already thinking of adding a second parallel FET, as our current max power would probably require heatsink contact with the FETs if we only use one per phase. Switching losses tend to increase linearly with each additional FET, but it is mostly spread across them (with some additional stress on the controller itself). Adding more than 2 FETs in parallel per-phase, we will start to see diminishing returns on that front. Adding more input voltage will also decrease our duty cycle. This will tend to decrease the input ripple, but will have the inverse effect on the output. That could require larger inductors, more output caps, and the associated power loss in those parts.

My current estimate is that a dual 7448 card with my current VDD design's power loss will pull about 80W. Running from 28v would likely increase that to 90W or so. That would probably be counteracted by the decreased losses on the Cube's 5v rail though. That power loss does mean that we would have to find a way of attaching the 28v rail to the cube with a connection that can handle 4A. Possibly even 5A if we want to account for those that run their Cube from a 24v supply as I've seen some people on the 68kmla forums discuss.

I think the biggest challenge in making the CPU run off of 28v would be the isolation circuity though. We would need to guarantee that the 28v rail never touched the 5v input. We would also need to ensure that the VDD rail doesn't kick on unless 5v and 3.3v are live, as the cube just applies 28v power the moment it is plugged in. At least according to their datasheets, 744x CPUs are rather picky about their power up sequence.

I know I'm getting a little long-winded at this point, but I do think it is worth mentioning that I'm also actively working on a new VRM for the cube that I plan to open source. I have the schematic nearly finished and will soon be ordering boards to build a bench test setup for it. It will have the capability to provide up to 23.5A on the 5v rail. One limitation there is that the connector on the Cube VRM itself is only rated at 3A per-pin and the 5v rail has 5 of them, meaning that the connector may start to overheat if you draw more than 15A from it. The other 8.5A can be pulled safely from the Molex side. The ~16-17A that this CPU card will be capable of pulling from 5v at full tilt would require supplemental power on a Cube regardless, at least  if we care about staying within the VRM connector's current rating.

Finally, good to know that you are cool hand soldering some of the larger components. I was really struggling to find good al-poly caps on LCSC and knowing that I can use nicer inductors with better consistency will help guarantee better current monitoring support. I'll make sure that all of the small passives that are annoying to solder can be found from LCSC though.

**Comment by dosdude1** [comment]
Yep, small passives are the main thing I definitely want done via assembly, as I don't want to sit there for hours soldering a bunch of small resistors and capacitors haha.

Seeing you're working on a custom Cube VRM, it may be best to just forget the idea of producing VCORE from the 28V PSU input, as seems to me as though it wouldn't be worth it, due to the issues you've described and the fact that it would require more components/routing in an already limited board space. We can of course add an extra power connection to tap off of Molex for extra current capacity on the 5V rail for Cubes.

**Comment by evanboonie** [comment]
Okay, I think I have the core components required for the VDD rail picked out. Here is a [DigiKey list](https://www.digikey.com/en/mylists/list/NV8N82KV5V) with everything in it. Thankfully, aluminum polymer capacitors seem to have improved since Sonnet built their card. I was able to reduce the overall component count and footprint by quite a bit, which will hopefully save us the board space needed for things like supplemental power connectors and DIP switches. The following need to be taken into account here:

- The 4 330uF al-poly caps are for the input.
- 6 of the 47uF ceramic caps are for the input and 6 are for the output.
- The two 0.33uF ceramic caps are for the input. Place these as close to the FET inputs as possible.
- We only need one set of 82 for the 0.1uF ceramic caps; one per VDD pin on each CPU. I put in two options, one in an 0201 package and one in 0403. These need to be as close to the VDD pins as possible, which will be easier to do with the 0201 package. However, 0201 is basically impossible to solder by hand and only extremely expensive pick and place machines can even use them. JLCPCB charges more, but will use them. Going with the 0403 would make it easier for others to manufacture these cards without paying for assembly or simply wanting to use their own machinery. We will probably find out more about which one we can use when designing the PCB.
- The 680uf al-poly caps are for the output.
- We should definitely use 1oz copper on the inner layers. Going thicker will probably make our minimum trace size too large to properly route a 360pin BGA part, but going thinner doesn't have any benefits besides lower cost (at least at JLC).

Based on my calculations, this design should allow huge load transients (up to 35A, when the CPU rockets from sleep, to prime95, and back to sleep) with 1-2% of voltage transient on the output with a very flat load line. In simpler terms: It will be great for overclocking. I still need to pick out all of the configuration passives and will update when I have those picked out. Once we have everything picked out, I'll definitely need to build some bench testing boards so I can hook this up to a scope to make sure it performs in real life like the math says it will.

I don't have the $5,000 oscilloscope and $5,000 power rail probe required to really test these kinds of things and it isn't quite in the cards for me to acquire those things right now. I have a 40A bench supply and a 40A electronic load, but I think I can supplement with one of those cheap 25A electronic loads that you can find on Amazon. I'm thinking of using the cheap one for baseline/sleep load emulation and just manage the transient jumps on my nice one. My cheaper scope should still get us pretty close. Best case it reports ripple as slightly worse than it actually is. I won't really be able to determine if we accidentally built a radio jammer, but that is relatively unlikely. If we can find someone with nicer test gear to help out, that might be an option if we do run into issues I'm unable to resolve with my current test gear.

**Comment by dosdude1** [comment]
The pitch of these BGA360 CPUs is wide enough that using 0201s aren't necessary. I'd prefer to stay away from 0201s. 0402s should be more than small enough to fit in areas between VIAs laid out underneath the CPU package, so I'm not worried about that. Are you sure one per VDD pin is necessary? Looking at the Sonnet card, it looks like they used about 6 or so per CPU, and similar is true with the 7450 CPU board I was looking at to derive the pinout of the MPx connector. For the rest of the components, I'd prefer 0603 or 0805, or larger if needed.

As for the testing you describe, I have a decent 400 MHz HP scope, though don't have a bench supply capable of delivering that much current. If absolutely necessary though, I do know someone who works for Keysight, and has literally millions of dollars with of Keysight test equipment at his disposal, who I could get to help me do any testing needed.

**Comment by dosdude1** [comment]
I've attached here a 7450-based PowerBook G4 schematic that I've been using for reference. This should provide a lot of good info about loads and power sequencing you may need.

[Apple PowerBook G 4 M 8407 550 MHz 820-1263 051-6223 MLB PBG4 Rev A Schematic Diagram.pdf](https://github.com/dosdude1/MPx-CPU-Board-744x/files/12693199/Apple.PowerBook.G.4.M.8407.550.MHz.820-1263.051-6223.MLB.PBG4.Rev.A.Schematic.Diagram.pdf)

**Comment by evanboonie** [comment]
I'm not positive that one 0.1uF cap is actually needed for every VDD pin, but that is what the 7448 datasheet recommends. The datasheet also recommends one per OVDD pin, which means 52 more for those. The Cisco NPE-G2 has 79 caps below its single 1.7Ghz 7448 for what that is worth. Thanks for the schematic. I've opened a new issue for the OVDD rail and have posted some initial findings on it there. In short, I think we really just have to make sure that OVDD starts up first and shuts down last.

**Comment by dosdude1** [comment]
Yep, according to PBG4 schematic, OVDD should come up around 7mS before VDD.

**Comment by evanboonie** [comment]
Getting OVDD to start up first will be easy. Just have it ramp fast and feed the 3.3v line into the enable input on the controller for VDD. Then OVDD should start up first every time, especially if we tune a slower soft start on the VDD rail. Shutdown is the more difficult part to design for. In general, the load on VDD should be much higher, meaning its caps will drain faster than the ones on OVDD and it will turn off more quickly. That might have something to do with the large tantalum cap that seems to be sitting off by itself on the Sonnet board under the "SM2A" chip (which I'm assuming is its OVDD regulator).

**Comment by evanboonie** [comment]
Come to think of it, if I get my custom Cube VRM done before I get a CPU board in hand (which seems likely right now), I could actually build an adapter PCB to use it as the power supply when load testing the CPU board. True integration testing haha.

**Comment by evanboonie** [comment]
On the topic of the 0.1uF caps, I do have a way that I think we could get away with fewer. I mapped out all of the power and ground pins on the 7448 datasheet and found that most of the VDD pins are diagonally adjacent to each other. If we connect multiple of the adjacent VDD vias to a single cap, we could have them share it. The caps should still connect to ground as close to the VDD and ground vias as possible. Theoretically, all VDD pins can be tied together in this way, but I'm skeptical about only using a single cap. We aren't so lucky on OVDD, as those pins are spread around the chip pretty evenly and are all right next to their own dedicated ground pin. Best case I say we follow the datasheet, but I know that placing 134 0402 caps could be annoying or outright impossible. I think we should try to use as many caps as we can though. At least these can be assembled by JLC.

I also noticed that on the 7400 boards, Apple seems to use 26 caps on the bottom of each CPU. The 7400 chips have 32 VDD/OVDD pins; so Apple used slightly fewer, but Cisco used even more than recommended.

**Comment by evanboonie** [comment]
I have updated the [DigiKey list](https://www.digikey.com/en/mylists/list/NV8N82KV5V) with the rest of the passive components. I haven't looked up equivalent parts on LCSC yet, but I tried to pick really common values where possible. It should be relatively easy to find them if LCSC doesn't have those exact parts in stock. A few of these are redundant, but will be needed for testing so we can optimize some components of the design on the bench. The testing components are as follows:

Load Line:
     The "load line" is referring to how much voltage drop we are willing to allow from low load to max load. There must be some level of voltage drop so that when the CPU releases the load when scaling back executions (or going to sleep), a voltage spike from the converter is avoided. Based on the 744x datasheets, the best I can determine is that we want to target a 50mV or better drop. I think our capacitance can do a pretty shallow drop, but I picked components for the following:
- 25mv drop: 24kO resistor and 47pF capacitor
- 50mv drop: 12kO resistor and 100pF capacitor
- 75mv drop: 8kO resistor and 150pF capacitor
- 100mv drop: 6kO resistor and 220pF capacitor

Soft Start:
    Soft start can be tuned with either a 100kO, 150kO, or 200kO resistor. Larger resistor == longer soft start time.

In addition to the above, the following also need taken into consideration:
- The 402O and one of the 100O resistors should be configured as a voltage divider to generate the 1V needed for VID selection and phase count selection.
- The OSRSEL pin on the controller will need options to jump to GND, VREF, and 3.3v. Once we determine the proper selection in testing, we can go back to a single jump point. I picked an 0603 jumper resistor for this purpose.
- The PCNT pin will need jumpers to the 1V divider (for two phase operation) and 5V (for single phase operation)
- The GFB/GNDSNS and VFB pins need options for 100O resistors. GFB to GND and VFB to VDD/out. This is for testing the boards with no CPUs installed and will not be necessary on the final design.
- The thermistor should be placed in the hottest part of the board. If this hits 85C, the controller will shut itself down for safety.
- If possible, we should avoid placing components between the two CPUs. This will make it easier to attach an adapter board to load test the power supply without having to install real CPUs.
- I selected 0805 resistors for VID and PLL. That should make it easier to build the PCB and/or reconfigure them. If we use a DIP switch for PLL, we can probably shrink these to 0603 to save space.
- I found a couple of DIP switches that should fit nicely to control the CPU multiplier and VID. I don't think we should include a DIP switch for VID in the final design as it would make it very easy to accidentally fry the CPU(s). I do think it would be very nice to have in the power rail testing phase though. 

I also added components for OVDD. I have another [issue discussing that in more detail](https://github.com/dosdude1/MPx-CPU-Board-744x/issues/2).

I also have an idea about the general PCB design that might make it easier to build the boards. If we can place most of the smaller passives on the bottom of the board and only have JLC build that half, we could simply use a stencil and solder paste on the top of the boards.

I've never used KiCad before, but I'll try to learn it so I can start working on the schematic.
